### PR TITLE
Update instructions for Apple-Pi Baker

### PR DESCRIPTION
**Pull request by kiresp**
Just ran through this process on the Mac and it was so much easier than anything listed earlier.

### CODE REVIEW

File: installation/installing-images/mac.md [hunk 1]
```diff
@@ -2,6 +2,15 @@
 
 On Mac OS you have the choice of the command line `dd` tool or using the graphical tool ImageWriter to write the image to your SD card.
 
+## Completely graphical interface
+- Connect the SD card reader with the SD card inside.
+- Install ApplePi-Baker from http://www.tweaking4all.com/hardware/raspberry-pi/macosx-apple-pi-baker/
+- Select device from list (Pi-Crust)
+- Prep the card for NOOBS (Prepare)
```
Comment (lurch) [inline installation/installing-images/mac.md]: I've not used Apple-Pi--Baker myself, but I believe if you're only writing an image file, then there's no need to "prep for NOOBS"

File: installation/installing-images/mac.md [hunk 2]
```diff
@@ -2,6 +2,15 @@
...
+- Select the IMG file you downloaded earlier
+- Click Resotre Backup
```
Comment (lurch) [inline installation/installing-images/mac.md]: Typo

**Comment by dliloch** [comment]
It will write the image directly from a zip file and it is fast.

**Comment by JamesH65** [comment]
I this PR has been superceded by the use of Etcher. Closing

**Pull request closed without merging**